benchmarks/dynamic_config_envoy_proxy.py

```diff
@@ -0,0 +1,102 @@
+"""Customized fixture to dynamically configure an Envoy Proxy."""
+
+import pytest
+import logging
+import os
+from typing import Generator
+
+from test.integration.integration_test_fixtures import determineIpVersionsFromEnvironment
+from envoy_proxy import EnvoyProxyServer, InjectHttpProxyIntegrationTestBase
```

Comment:
how do you want to handle pytest fixtures conflicting with this rule.

e.g. 

```
\@pytest.fixture()
def my_fixture()
   yield someFooFixture()
```

but in another module we have to import this function otherwise pytest would pick it up.

Reply:
done for added code apart from the conflict mentioned above.

Reply:
Fair point about pytest fixtures, I think it is fine to exclude those from this rule.

Sorry for not realizing that, I am not used to pytest yet.